files/es/web/api/crypto/getrandomvalues/index.html

```diff
@@ -29,7 +29,7 @@ <h3 id="Parámetros">Parámetros</h3>
 <h3 id="Excepciones">Excepciones</h3>
 
 <ul>
- <li>Una {{exception("QuotaExceededError")}} {{domxref("DOMException")}} es lanzada si la longitud solicitada es mayor a 65536 bytes.</li>
+ <li>Una {{domxref("DOMException")}} es lanzada si la longitud solicitada es mayor a 65536 bytes.</li>
```

Comment:
Veo que si afecta al contenido mostrado

```suggestion
 <li>Una <a href="/es/docs/Web/API/DOMException#quotaexceedederror">QuotaExceededError</a> {{domxref("DOMException")}} es lanzada si la longitud solicitada es mayor a 65536 bytes.</li>
```

Reply:
Vi en otros lenguajes que removieron el `exception('QuataExceededError')` sin agregar el link, por eso pense que es lo correcto. ahora que hacemos?

Reply:
Justo vi en `fr` el ejemplo y en `closest` de este PR también lo sustituiste por el link